frontend/src/metabase/components/HistoryModal.jsx

```diff
@@ -34,10 +34,27 @@ export default class HistoryModal extends Component {
     }
   }
 
+  shouldRenderRevisionEntry({ diff }) {
+    // diff may be null in "First revision", that is
+    // initial state.
+    if (diff === null) {
+      return true;
+    }
+
+    const { before, after } = diff;
+    return before !== null && after !== null;
```

Comment:
i think this makes me nervous. I think if either is different we should show it. An example is removing a description:

```clojure
;; {:description "something"} -> {}
(default-diff-map nil {:description "something"} {})
{:before {:description "something"}, :after nil}
```

Reply:
@dpsutton updated, thank you.

This time around I will actively ask Nemanja for a look at the e2e test before merging. It's my first PR.

Happy to improve anything you may see there.

Thanks for the review!